common/logger.go

```diff
@@ -9,6 +9,7 @@ import (
 var Logger LogAdapter
 
 type LogAdapter interface {
+	Dev(m string)
```

Comment:
Does this need to be part of the interface?  It looks like `DevLogger` is the only one that uses it